sickbeard/server/web/core/base.py

```diff
@@ -91,6 +91,10 @@ def __init__(self, rh, filename):
             'newsBadge': '',
             'toolsBadge': '',
             'toolsBadgeClass': '',
+            'base_url': urljoin(rh.request.protocol + '://' + rh.request.host,
+                                sickbeard.WEB_ROOT) +
+                          ('', '/')
+                          [urljoin(rh.request.protocol + '://' + rh.request.host, sickbeard.WEB_ROOT)[-1] not in '/'],
```

Comment:
Why would web root change? 
The issue is that when we don't have Web_root set, we add an ending /. 
So we either have / or /webroot
In case we don't have Web_root set, it would cause base_url to have a double /.